### ISSUE TITLE
[Feature] Aid Another (Interfere)

### ISSUE THREAD

**Issue by DoctorBak** [GrognardsFromHell/TemplePlus#246]
Although the Aid Another action has already been fixed (as reported in the changelog) it is limited to the wake-up mechanic. The AA entry in the TOEE manual describes a further application: a friendly target gains a +2 (arguably a circumstance bonus) to his attack rolls until your next turn.

Assuming the will, this feature could be added in 2 ways:
- applying the TOEE manual definition, which could make AA a more powerful tactic than the D&D version.
- applying the SRD definition, which could be harder to implement (requires to select two targets, choose between two bonuses, roll an attack).

Another way is to opt for a middle ground: Interfere (standard action)
- select an enemy you threaten
- roll an attack against AC 10
- if succeeded, the enemy has -2 to his attack rolls and AC
- the effect lasts until the enemy attacks/receives an attack/your next turn, whichever comes first.

The outcome is pretty much the same (slightly more favourable) but is easier to break it down into code i guess.

These are just my impressions, you may argue that the Aid Another feature is not a fundamental one and i would struggle to disagree, though in TOEE every tool happens to be crucial for survival.

**Comment by DudeMcDude** [comment]
Yeah it's not a high priority since I think most people would rather just attack with that character. But it's on the list along with the defensive assist.
